src/reflow/mod.rs

```diff
@@ -68,21 +68,21 @@ fn reflow_inner<'s>(
     let mut lines = s_absolute.lines();
     let mut indents = indentations.iter();
     let last_ident =
-        indentations.last().expect("Must contain one indentation") - variant.prefix_len();
+        indentations.last().expect("Must contain one indentation").saturating_sub(variant.prefix_len());
     // first line has to be without indent and variant prefix
     let (_, content, _) = gluon.next().expect("Must contain one line");
-    if lines.next() != Some(&content) {
+    let mut reflow_applied = lines.next().filter_map(|linecontent| content == linecontent).is_none();
```

Comment:
Missing openening brackets I guess? Mea culpa.

Reply:
Actually we can't use that at all, `next()` returns an Option. I reverted back to the old solution, what do you suggest?

Reply:
Hmm, I thought there was ,`map` and `filter` both exist :man_shrugging: